src/common/style/base.css

```diff
@@ -62,7 +62,7 @@ a:focus {
 }
 
 body {
-  background-color: $light-grey;
+  background-color: #FFF;
```

Comment:
Before we used styles from MU that had body background as light grey, but new designs have white background.

This affects all the pages.